problem-solver/py/services/kindergartensModule/GetKindergartensByStreetAgent.py

```diff
@@ -0,0 +1,114 @@
+from termcolor import colored
+
+from common import ScAgent, ScEventParams, ScKeynodes
+from sc import *
+
+
+class GetKindergartensByStreetAgent(ScAgent):
+    def __init__(self, module):
+        super().__init__(module)
+        self.ctx = module.ctx
+        self.keynodes = ScKeynodes(self.ctx)
+        self.main_node = None
+
+    def RunImpl(self, evt: ScEventParams) -> ScResult:
+        self.main_node = evt.other_addr
+        status = ScResult.Ok
+
+        if self.module.ctx.HelperCheckEdge(
+                self.keynodes['action_get_kindergartens_by_street'],
```

Comment:
в ui_menu файле для этого агента вы прописали action так, перепроверьте у всех агентов эти строки
```suggestion
                self.keynodes['action_get_kindergarden_by_street'],
```